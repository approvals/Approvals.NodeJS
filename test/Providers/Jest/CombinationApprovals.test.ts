import {describe, test} from "@jest/globals";
import {
    verifyAllCombinations1,
    verifyAllCombinations2,
    verifyAllCombinations3,
    verifyAllCombinations4,
    verifyAllCombinations5,
    verifyAllCombinations6,
    verifyAllCombinations7, verifyAllCombinations8, verifyAllCombinations9
} from "../../../lib/Providers/Jest/CombinationApprovals";


describe("Combinations", () => {
    test("verify1", () => {
        function binarySquare(i: number) {
            if (i === 2) {
                throw new Error("There is no 2");
            }
            return i;
        }

        const numbers = [0, 1, 2];
        verifyAllCombinations1(binarySquare, numbers);
    });
    test("verify2", () => {
        const numbers = [0, 1, 2];
        verifyAllCombinations2((a, b) => a + b, numbers, numbers);
    });
    test("verify3", () => {
        const numbers = [3, 4];
        verifyAllCombinations3((a, b, c) => a * b * c, numbers, numbers, numbers);
    });
    test("verify4", () => {
        const numbers = [3, 4];
        verifyAllCombinations4((a, b, c, d) => a + "" + b + c + d, numbers, numbers, numbers, numbers);
    });
    test("verify5", () => {
        verifyAllCombinations5((a, b, c, d, e) => e + d + c + b + a, ["a", "b"], ["a", "b"], ["c", "d"], ["e", "f"], ["g", "h", "i"]);
    });
    test("verify6", () => {
        verifyAllCombinations6((a, b, c, d, e, f) => "#" + a + " " + f + e + d + c + b, [1, 2], ["a", "b"], ["c", "d"], ["e", "f"], ["i", "j"], ["g", "h"]);
    });
    test("verify7", () => {
        verifyAllCombinations7((a, b, c, d, e, f, g) => a + b + c + d + e + f + g, [1, 2], [2, 3], [2, 3], [2, 3], [2, 3], [2, 3], [2, 3]);
    });
    test("verify8", () => {
        verifyAllCombinations8((a, b, c, d, e, f, g, h) => a + b + c + d + e + f + g + h, [1, 2], [2, 3], [2, 3], [2, 3], [2, 3], [2, 3], [2, 3], [8, 7]);
    });
    test("verify9", () => {
        verifyAllCombinations9((a, b, c, d, e, f, g, h, i) => "" + a + b + c + d + e + f + g + h + i, [1, 2], [2, 3], [2, 3], [2, 3], [2, 3], [2, 3], [2, 3], [8, 7], [9, 8]);
    });
});

describe('documentation', () => {
    // begin-snippet: CombinationsStartingPoint
    test('CombinationsStartingPoint', () => {
        const inputs1 = ["input1.value1", "input1.value2"];
        const inputs2 = ["input2.value1", "input2.value2", "input2.value3"];
        verifyAllCombinations2((a, b) => `placeholder for ${a}, ${b}`, inputs1, inputs2);
    });
    // end-snippet
});
