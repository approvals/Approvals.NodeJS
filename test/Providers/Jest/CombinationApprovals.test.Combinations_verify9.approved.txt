[1,2,2,2,2,2,2,8,9] => 122222289
[1,2,2,2,2,2,2,8,8] => 122222288
[1,2,2,2,2,2,2,7,9] => 122222279
[1,2,2,2,2,2,2,7,8] => 122222278
[1,2,2,2,2,2,3,8,9] => 122222389
[1,2,2,2,2,2,3,8,8] => 122222388
[1,2,2,2,2,2,3,7,9] => 122222379
[1,2,2,2,2,2,3,7,8] => 122222378
[1,2,2,2,2,3,2,8,9] => 122223289
[1,2,2,2,2,3,2,8,8] => 122223288
[1,2,2,2,2,3,2,7,9] => 122223279
[1,2,2,2,2,3,2,7,8] => 122223278
[1,2,2,2,2,3,3,8,9] => 122223389
[1,2,2,2,2,3,3,8,8] => 122223388
[1,2,2,2,2,3,3,7,9] => 122223379
[1,2,2,2,2,3,3,7,8] => 122223378
[1,2,2,2,3,2,2,8,9] => 122232289
[1,2,2,2,3,2,2,8,8] => 122232288
[1,2,2,2,3,2,2,7,9] => 122232279
[1,2,2,2,3,2,2,7,8] => 122232278
[1,2,2,2,3,2,3,8,9] => 122232389
[1,2,2,2,3,2,3,8,8] => 122232388
[1,2,2,2,3,2,3,7,9] => 122232379
[1,2,2,2,3,2,3,7,8] => 122232378
[1,2,2,2,3,3,2,8,9] => 122233289
[1,2,2,2,3,3,2,8,8] => 122233288
[1,2,2,2,3,3,2,7,9] => 122233279
[1,2,2,2,3,3,2,7,8] => 122233278
[1,2,2,2,3,3,3,8,9] => 122233389
[1,2,2,2,3,3,3,8,8] => 122233388
[1,2,2,2,3,3,3,7,9] => 122233379
[1,2,2,2,3,3,3,7,8] => 122233378
[1,2,2,3,2,2,2,8,9] => 122322289
[1,2,2,3,2,2,2,8,8] => 122322288
[1,2,2,3,2,2,2,7,9] => 122322279
[1,2,2,3,2,2,2,7,8] => 122322278
[1,2,2,3,2,2,3,8,9] => 122322389
[1,2,2,3,2,2,3,8,8] => 122322388
[1,2,2,3,2,2,3,7,9] => 122322379
[1,2,2,3,2,2,3,7,8] => 122322378
[1,2,2,3,2,3,2,8,9] => 122323289
[1,2,2,3,2,3,2,8,8] => 122323288
[1,2,2,3,2,3,2,7,9] => 122323279
[1,2,2,3,2,3,2,7,8] => 122323278
[1,2,2,3,2,3,3,8,9] => 122323389
[1,2,2,3,2,3,3,8,8] => 122323388
[1,2,2,3,2,3,3,7,9] => 122323379
[1,2,2,3,2,3,3,7,8] => 122323378
[1,2,2,3,3,2,2,8,9] => 122332289
[1,2,2,3,3,2,2,8,8] => 122332288
[1,2,2,3,3,2,2,7,9] => 122332279
[1,2,2,3,3,2,2,7,8] => 122332278
[1,2,2,3,3,2,3,8,9] => 122332389
[1,2,2,3,3,2,3,8,8] => 122332388
[1,2,2,3,3,2,3,7,9] => 122332379
[1,2,2,3,3,2,3,7,8] => 122332378
[1,2,2,3,3,3,2,8,9] => 122333289
[1,2,2,3,3,3,2,8,8] => 122333288
[1,2,2,3,3,3,2,7,9] => 122333279
[1,2,2,3,3,3,2,7,8] => 122333278
[1,2,2,3,3,3,3,8,9] => 122333389
[1,2,2,3,3,3,3,8,8] => 122333388
[1,2,2,3,3,3,3,7,9] => 122333379
[1,2,2,3,3,3,3,7,8] => 122333378
[1,2,3,2,2,2,2,8,9] => 123222289
[1,2,3,2,2,2,2,8,8] => 123222288
[1,2,3,2,2,2,2,7,9] => 123222279
[1,2,3,2,2,2,2,7,8] => 123222278
[1,2,3,2,2,2,3,8,9] => 123222389
[1,2,3,2,2,2,3,8,8] => 123222388
[1,2,3,2,2,2,3,7,9] => 123222379
[1,2,3,2,2,2,3,7,8] => 123222378
[1,2,3,2,2,3,2,8,9] => 123223289
[1,2,3,2,2,3,2,8,8] => 123223288
[1,2,3,2,2,3,2,7,9] => 123223279
[1,2,3,2,2,3,2,7,8] => 123223278
[1,2,3,2,2,3,3,8,9] => 123223389
[1,2,3,2,2,3,3,8,8] => 123223388
[1,2,3,2,2,3,3,7,9] => 123223379
[1,2,3,2,2,3,3,7,8] => 123223378
[1,2,3,2,3,2,2,8,9] => 123232289
[1,2,3,2,3,2,2,8,8] => 123232288
[1,2,3,2,3,2,2,7,9] => 123232279
[1,2,3,2,3,2,2,7,8] => 123232278
[1,2,3,2,3,2,3,8,9] => 123232389
[1,2,3,2,3,2,3,8,8] => 123232388
[1,2,3,2,3,2,3,7,9] => 123232379
[1,2,3,2,3,2,3,7,8] => 123232378
[1,2,3,2,3,3,2,8,9] => 123233289
[1,2,3,2,3,3,2,8,8] => 123233288
[1,2,3,2,3,3,2,7,9] => 123233279
[1,2,3,2,3,3,2,7,8] => 123233278
[1,2,3,2,3,3,3,8,9] => 123233389
[1,2,3,2,3,3,3,8,8] => 123233388
[1,2,3,2,3,3,3,7,9] => 123233379
[1,2,3,2,3,3,3,7,8] => 123233378
[1,2,3,3,2,2,2,8,9] => 123322289
[1,2,3,3,2,2,2,8,8] => 123322288
[1,2,3,3,2,2,2,7,9] => 123322279
[1,2,3,3,2,2,2,7,8] => 123322278
[1,2,3,3,2,2,3,8,9] => 123322389
[1,2,3,3,2,2,3,8,8] => 123322388
[1,2,3,3,2,2,3,7,9] => 123322379
[1,2,3,3,2,2,3,7,8] => 123322378
[1,2,3,3,2,3,2,8,9] => 123323289
[1,2,3,3,2,3,2,8,8] => 123323288
[1,2,3,3,2,3,2,7,9] => 123323279
[1,2,3,3,2,3,2,7,8] => 123323278
[1,2,3,3,2,3,3,8,9] => 123323389
[1,2,3,3,2,3,3,8,8] => 123323388
[1,2,3,3,2,3,3,7,9] => 123323379
[1,2,3,3,2,3,3,7,8] => 123323378
[1,2,3,3,3,2,2,8,9] => 123332289
[1,2,3,3,3,2,2,8,8] => 123332288
[1,2,3,3,3,2,2,7,9] => 123332279
[1,2,3,3,3,2,2,7,8] => 123332278
[1,2,3,3,3,2,3,8,9] => 123332389
[1,2,3,3,3,2,3,8,8] => 123332388
[1,2,3,3,3,2,3,7,9] => 123332379
[1,2,3,3,3,2,3,7,8] => 123332378
[1,2,3,3,3,3,2,8,9] => 123333289
[1,2,3,3,3,3,2,8,8] => 123333288
[1,2,3,3,3,3,2,7,9] => 123333279
[1,2,3,3,3,3,2,7,8] => 123333278
[1,2,3,3,3,3,3,8,9] => 123333389
[1,2,3,3,3,3,3,8,8] => 123333388
[1,2,3,3,3,3,3,7,9] => 123333379
[1,2,3,3,3,3,3,7,8] => 123333378
[1,3,2,2,2,2,2,8,9] => 132222289
[1,3,2,2,2,2,2,8,8] => 132222288
[1,3,2,2,2,2,2,7,9] => 132222279
[1,3,2,2,2,2,2,7,8] => 132222278
[1,3,2,2,2,2,3,8,9] => 132222389
[1,3,2,2,2,2,3,8,8] => 132222388
[1,3,2,2,2,2,3,7,9] => 132222379
[1,3,2,2,2,2,3,7,8] => 132222378
[1,3,2,2,2,3,2,8,9] => 132223289
[1,3,2,2,2,3,2,8,8] => 132223288
[1,3,2,2,2,3,2,7,9] => 132223279
[1,3,2,2,2,3,2,7,8] => 132223278
[1,3,2,2,2,3,3,8,9] => 132223389
[1,3,2,2,2,3,3,8,8] => 132223388
[1,3,2,2,2,3,3,7,9] => 132223379
[1,3,2,2,2,3,3,7,8] => 132223378
[1,3,2,2,3,2,2,8,9] => 132232289
[1,3,2,2,3,2,2,8,8] => 132232288
[1,3,2,2,3,2,2,7,9] => 132232279
[1,3,2,2,3,2,2,7,8] => 132232278
[1,3,2,2,3,2,3,8,9] => 132232389
[1,3,2,2,3,2,3,8,8] => 132232388
[1,3,2,2,3,2,3,7,9] => 132232379
[1,3,2,2,3,2,3,7,8] => 132232378
[1,3,2,2,3,3,2,8,9] => 132233289
[1,3,2,2,3,3,2,8,8] => 132233288
[1,3,2,2,3,3,2,7,9] => 132233279
[1,3,2,2,3,3,2,7,8] => 132233278
[1,3,2,2,3,3,3,8,9] => 132233389
[1,3,2,2,3,3,3,8,8] => 132233388
[1,3,2,2,3,3,3,7,9] => 132233379
[1,3,2,2,3,3,3,7,8] => 132233378
[1,3,2,3,2,2,2,8,9] => 132322289
[1,3,2,3,2,2,2,8,8] => 132322288
[1,3,2,3,2,2,2,7,9] => 132322279
[1,3,2,3,2,2,2,7,8] => 132322278
[1,3,2,3,2,2,3,8,9] => 132322389
[1,3,2,3,2,2,3,8,8] => 132322388
[1,3,2,3,2,2,3,7,9] => 132322379
[1,3,2,3,2,2,3,7,8] => 132322378
[1,3,2,3,2,3,2,8,9] => 132323289
[1,3,2,3,2,3,2,8,8] => 132323288
[1,3,2,3,2,3,2,7,9] => 132323279
[1,3,2,3,2,3,2,7,8] => 132323278
[1,3,2,3,2,3,3,8,9] => 132323389
[1,3,2,3,2,3,3,8,8] => 132323388
[1,3,2,3,2,3,3,7,9] => 132323379
[1,3,2,3,2,3,3,7,8] => 132323378
[1,3,2,3,3,2,2,8,9] => 132332289
[1,3,2,3,3,2,2,8,8] => 132332288
[1,3,2,3,3,2,2,7,9] => 132332279
[1,3,2,3,3,2,2,7,8] => 132332278
[1,3,2,3,3,2,3,8,9] => 132332389
[1,3,2,3,3,2,3,8,8] => 132332388
[1,3,2,3,3,2,3,7,9] => 132332379
[1,3,2,3,3,2,3,7,8] => 132332378
[1,3,2,3,3,3,2,8,9] => 132333289
[1,3,2,3,3,3,2,8,8] => 132333288
[1,3,2,3,3,3,2,7,9] => 132333279
[1,3,2,3,3,3,2,7,8] => 132333278
[1,3,2,3,3,3,3,8,9] => 132333389
[1,3,2,3,3,3,3,8,8] => 132333388
[1,3,2,3,3,3,3,7,9] => 132333379
[1,3,2,3,3,3,3,7,8] => 132333378
[1,3,3,2,2,2,2,8,9] => 133222289
[1,3,3,2,2,2,2,8,8] => 133222288
[1,3,3,2,2,2,2,7,9] => 133222279
[1,3,3,2,2,2,2,7,8] => 133222278
[1,3,3,2,2,2,3,8,9] => 133222389
[1,3,3,2,2,2,3,8,8] => 133222388
[1,3,3,2,2,2,3,7,9] => 133222379
[1,3,3,2,2,2,3,7,8] => 133222378
[1,3,3,2,2,3,2,8,9] => 133223289
[1,3,3,2,2,3,2,8,8] => 133223288
[1,3,3,2,2,3,2,7,9] => 133223279
[1,3,3,2,2,3,2,7,8] => 133223278
[1,3,3,2,2,3,3,8,9] => 133223389
[1,3,3,2,2,3,3,8,8] => 133223388
[1,3,3,2,2,3,3,7,9] => 133223379
[1,3,3,2,2,3,3,7,8] => 133223378
[1,3,3,2,3,2,2,8,9] => 133232289
[1,3,3,2,3,2,2,8,8] => 133232288
[1,3,3,2,3,2,2,7,9] => 133232279
[1,3,3,2,3,2,2,7,8] => 133232278
[1,3,3,2,3,2,3,8,9] => 133232389
[1,3,3,2,3,2,3,8,8] => 133232388
[1,3,3,2,3,2,3,7,9] => 133232379
[1,3,3,2,3,2,3,7,8] => 133232378
[1,3,3,2,3,3,2,8,9] => 133233289
[1,3,3,2,3,3,2,8,8] => 133233288
[1,3,3,2,3,3,2,7,9] => 133233279
[1,3,3,2,3,3,2,7,8] => 133233278
[1,3,3,2,3,3,3,8,9] => 133233389
[1,3,3,2,3,3,3,8,8] => 133233388
[1,3,3,2,3,3,3,7,9] => 133233379
[1,3,3,2,3,3,3,7,8] => 133233378
[1,3,3,3,2,2,2,8,9] => 133322289
[1,3,3,3,2,2,2,8,8] => 133322288
[1,3,3,3,2,2,2,7,9] => 133322279
[1,3,3,3,2,2,2,7,8] => 133322278
[1,3,3,3,2,2,3,8,9] => 133322389
[1,3,3,3,2,2,3,8,8] => 133322388
[1,3,3,3,2,2,3,7,9] => 133322379
[1,3,3,3,2,2,3,7,8] => 133322378
[1,3,3,3,2,3,2,8,9] => 133323289
[1,3,3,3,2,3,2,8,8] => 133323288
[1,3,3,3,2,3,2,7,9] => 133323279
[1,3,3,3,2,3,2,7,8] => 133323278
[1,3,3,3,2,3,3,8,9] => 133323389
[1,3,3,3,2,3,3,8,8] => 133323388
[1,3,3,3,2,3,3,7,9] => 133323379
[1,3,3,3,2,3,3,7,8] => 133323378
[1,3,3,3,3,2,2,8,9] => 133332289
[1,3,3,3,3,2,2,8,8] => 133332288
[1,3,3,3,3,2,2,7,9] => 133332279
[1,3,3,3,3,2,2,7,8] => 133332278
[1,3,3,3,3,2,3,8,9] => 133332389
[1,3,3,3,3,2,3,8,8] => 133332388
[1,3,3,3,3,2,3,7,9] => 133332379
[1,3,3,3,3,2,3,7,8] => 133332378
[1,3,3,3,3,3,2,8,9] => 133333289
[1,3,3,3,3,3,2,8,8] => 133333288
[1,3,3,3,3,3,2,7,9] => 133333279
[1,3,3,3,3,3,2,7,8] => 133333278
[1,3,3,3,3,3,3,8,9] => 133333389
[1,3,3,3,3,3,3,8,8] => 133333388
[1,3,3,3,3,3,3,7,9] => 133333379
[1,3,3,3,3,3,3,7,8] => 133333378
[2,2,2,2,2,2,2,8,9] => 222222289
[2,2,2,2,2,2,2,8,8] => 222222288
[2,2,2,2,2,2,2,7,9] => 222222279
[2,2,2,2,2,2,2,7,8] => 222222278
[2,2,2,2,2,2,3,8,9] => 222222389
[2,2,2,2,2,2,3,8,8] => 222222388
[2,2,2,2,2,2,3,7,9] => 222222379
[2,2,2,2,2,2,3,7,8] => 222222378
[2,2,2,2,2,3,2,8,9] => 222223289
[2,2,2,2,2,3,2,8,8] => 222223288
[2,2,2,2,2,3,2,7,9] => 222223279
[2,2,2,2,2,3,2,7,8] => 222223278
[2,2,2,2,2,3,3,8,9] => 222223389
[2,2,2,2,2,3,3,8,8] => 222223388
[2,2,2,2,2,3,3,7,9] => 222223379
[2,2,2,2,2,3,3,7,8] => 222223378
[2,2,2,2,3,2,2,8,9] => 222232289
[2,2,2,2,3,2,2,8,8] => 222232288
[2,2,2,2,3,2,2,7,9] => 222232279
[2,2,2,2,3,2,2,7,8] => 222232278
[2,2,2,2,3,2,3,8,9] => 222232389
[2,2,2,2,3,2,3,8,8] => 222232388
[2,2,2,2,3,2,3,7,9] => 222232379
[2,2,2,2,3,2,3,7,8] => 222232378
[2,2,2,2,3,3,2,8,9] => 222233289
[2,2,2,2,3,3,2,8,8] => 222233288
[2,2,2,2,3,3,2,7,9] => 222233279
[2,2,2,2,3,3,2,7,8] => 222233278
[2,2,2,2,3,3,3,8,9] => 222233389
[2,2,2,2,3,3,3,8,8] => 222233388
[2,2,2,2,3,3,3,7,9] => 222233379
[2,2,2,2,3,3,3,7,8] => 222233378
[2,2,2,3,2,2,2,8,9] => 222322289
[2,2,2,3,2,2,2,8,8] => 222322288
[2,2,2,3,2,2,2,7,9] => 222322279
[2,2,2,3,2,2,2,7,8] => 222322278
[2,2,2,3,2,2,3,8,9] => 222322389
[2,2,2,3,2,2,3,8,8] => 222322388
[2,2,2,3,2,2,3,7,9] => 222322379
[2,2,2,3,2,2,3,7,8] => 222322378
[2,2,2,3,2,3,2,8,9] => 222323289
[2,2,2,3,2,3,2,8,8] => 222323288
[2,2,2,3,2,3,2,7,9] => 222323279
[2,2,2,3,2,3,2,7,8] => 222323278
[2,2,2,3,2,3,3,8,9] => 222323389
[2,2,2,3,2,3,3,8,8] => 222323388
[2,2,2,3,2,3,3,7,9] => 222323379
[2,2,2,3,2,3,3,7,8] => 222323378
[2,2,2,3,3,2,2,8,9] => 222332289
[2,2,2,3,3,2,2,8,8] => 222332288
[2,2,2,3,3,2,2,7,9] => 222332279
[2,2,2,3,3,2,2,7,8] => 222332278
[2,2,2,3,3,2,3,8,9] => 222332389
[2,2,2,3,3,2,3,8,8] => 222332388
[2,2,2,3,3,2,3,7,9] => 222332379
[2,2,2,3,3,2,3,7,8] => 222332378
[2,2,2,3,3,3,2,8,9] => 222333289
[2,2,2,3,3,3,2,8,8] => 222333288
[2,2,2,3,3,3,2,7,9] => 222333279
[2,2,2,3,3,3,2,7,8] => 222333278
[2,2,2,3,3,3,3,8,9] => 222333389
[2,2,2,3,3,3,3,8,8] => 222333388
[2,2,2,3,3,3,3,7,9] => 222333379
[2,2,2,3,3,3,3,7,8] => 222333378
[2,2,3,2,2,2,2,8,9] => 223222289
[2,2,3,2,2,2,2,8,8] => 223222288
[2,2,3,2,2,2,2,7,9] => 223222279
[2,2,3,2,2,2,2,7,8] => 223222278
[2,2,3,2,2,2,3,8,9] => 223222389
[2,2,3,2,2,2,3,8,8] => 223222388
[2,2,3,2,2,2,3,7,9] => 223222379
[2,2,3,2,2,2,3,7,8] => 223222378
[2,2,3,2,2,3,2,8,9] => 223223289
[2,2,3,2,2,3,2,8,8] => 223223288
[2,2,3,2,2,3,2,7,9] => 223223279
[2,2,3,2,2,3,2,7,8] => 223223278
[2,2,3,2,2,3,3,8,9] => 223223389
[2,2,3,2,2,3,3,8,8] => 223223388
[2,2,3,2,2,3,3,7,9] => 223223379
[2,2,3,2,2,3,3,7,8] => 223223378
[2,2,3,2,3,2,2,8,9] => 223232289
[2,2,3,2,3,2,2,8,8] => 223232288
[2,2,3,2,3,2,2,7,9] => 223232279
[2,2,3,2,3,2,2,7,8] => 223232278
[2,2,3,2,3,2,3,8,9] => 223232389
[2,2,3,2,3,2,3,8,8] => 223232388
[2,2,3,2,3,2,3,7,9] => 223232379
[2,2,3,2,3,2,3,7,8] => 223232378
[2,2,3,2,3,3,2,8,9] => 223233289
[2,2,3,2,3,3,2,8,8] => 223233288
[2,2,3,2,3,3,2,7,9] => 223233279
[2,2,3,2,3,3,2,7,8] => 223233278
[2,2,3,2,3,3,3,8,9] => 223233389
[2,2,3,2,3,3,3,8,8] => 223233388
[2,2,3,2,3,3,3,7,9] => 223233379
[2,2,3,2,3,3,3,7,8] => 223233378
[2,2,3,3,2,2,2,8,9] => 223322289
[2,2,3,3,2,2,2,8,8] => 223322288
[2,2,3,3,2,2,2,7,9] => 223322279
[2,2,3,3,2,2,2,7,8] => 223322278
[2,2,3,3,2,2,3,8,9] => 223322389
[2,2,3,3,2,2,3,8,8] => 223322388
[2,2,3,3,2,2,3,7,9] => 223322379
[2,2,3,3,2,2,3,7,8] => 223322378
[2,2,3,3,2,3,2,8,9] => 223323289
[2,2,3,3,2,3,2,8,8] => 223323288
[2,2,3,3,2,3,2,7,9] => 223323279
[2,2,3,3,2,3,2,7,8] => 223323278
[2,2,3,3,2,3,3,8,9] => 223323389
[2,2,3,3,2,3,3,8,8] => 223323388
[2,2,3,3,2,3,3,7,9] => 223323379
[2,2,3,3,2,3,3,7,8] => 223323378
[2,2,3,3,3,2,2,8,9] => 223332289
[2,2,3,3,3,2,2,8,8] => 223332288
[2,2,3,3,3,2,2,7,9] => 223332279
[2,2,3,3,3,2,2,7,8] => 223332278
[2,2,3,3,3,2,3,8,9] => 223332389
[2,2,3,3,3,2,3,8,8] => 223332388
[2,2,3,3,3,2,3,7,9] => 223332379
[2,2,3,3,3,2,3,7,8] => 223332378
[2,2,3,3,3,3,2,8,9] => 223333289
[2,2,3,3,3,3,2,8,8] => 223333288
[2,2,3,3,3,3,2,7,9] => 223333279
[2,2,3,3,3,3,2,7,8] => 223333278
[2,2,3,3,3,3,3,8,9] => 223333389
[2,2,3,3,3,3,3,8,8] => 223333388
[2,2,3,3,3,3,3,7,9] => 223333379
[2,2,3,3,3,3,3,7,8] => 223333378
[2,3,2,2,2,2,2,8,9] => 232222289
[2,3,2,2,2,2,2,8,8] => 232222288
[2,3,2,2,2,2,2,7,9] => 232222279
[2,3,2,2,2,2,2,7,8] => 232222278
[2,3,2,2,2,2,3,8,9] => 232222389
[2,3,2,2,2,2,3,8,8] => 232222388
[2,3,2,2,2,2,3,7,9] => 232222379
[2,3,2,2,2,2,3,7,8] => 232222378
[2,3,2,2,2,3,2,8,9] => 232223289
[2,3,2,2,2,3,2,8,8] => 232223288
[2,3,2,2,2,3,2,7,9] => 232223279
[2,3,2,2,2,3,2,7,8] => 232223278
[2,3,2,2,2,3,3,8,9] => 232223389
[2,3,2,2,2,3,3,8,8] => 232223388
[2,3,2,2,2,3,3,7,9] => 232223379
[2,3,2,2,2,3,3,7,8] => 232223378
[2,3,2,2,3,2,2,8,9] => 232232289
[2,3,2,2,3,2,2,8,8] => 232232288
[2,3,2,2,3,2,2,7,9] => 232232279
[2,3,2,2,3,2,2,7,8] => 232232278
[2,3,2,2,3,2,3,8,9] => 232232389
[2,3,2,2,3,2,3,8,8] => 232232388
[2,3,2,2,3,2,3,7,9] => 232232379
[2,3,2,2,3,2,3,7,8] => 232232378
[2,3,2,2,3,3,2,8,9] => 232233289
[2,3,2,2,3,3,2,8,8] => 232233288
[2,3,2,2,3,3,2,7,9] => 232233279
[2,3,2,2,3,3,2,7,8] => 232233278
[2,3,2,2,3,3,3,8,9] => 232233389
[2,3,2,2,3,3,3,8,8] => 232233388
[2,3,2,2,3,3,3,7,9] => 232233379
[2,3,2,2,3,3,3,7,8] => 232233378
[2,3,2,3,2,2,2,8,9] => 232322289
[2,3,2,3,2,2,2,8,8] => 232322288
[2,3,2,3,2,2,2,7,9] => 232322279
[2,3,2,3,2,2,2,7,8] => 232322278
[2,3,2,3,2,2,3,8,9] => 232322389
[2,3,2,3,2,2,3,8,8] => 232322388
[2,3,2,3,2,2,3,7,9] => 232322379
[2,3,2,3,2,2,3,7,8] => 232322378
[2,3,2,3,2,3,2,8,9] => 232323289
[2,3,2,3,2,3,2,8,8] => 232323288
[2,3,2,3,2,3,2,7,9] => 232323279
[2,3,2,3,2,3,2,7,8] => 232323278
[2,3,2,3,2,3,3,8,9] => 232323389
[2,3,2,3,2,3,3,8,8] => 232323388
[2,3,2,3,2,3,3,7,9] => 232323379
[2,3,2,3,2,3,3,7,8] => 232323378
[2,3,2,3,3,2,2,8,9] => 232332289
[2,3,2,3,3,2,2,8,8] => 232332288
[2,3,2,3,3,2,2,7,9] => 232332279
[2,3,2,3,3,2,2,7,8] => 232332278
[2,3,2,3,3,2,3,8,9] => 232332389
[2,3,2,3,3,2,3,8,8] => 232332388
[2,3,2,3,3,2,3,7,9] => 232332379
[2,3,2,3,3,2,3,7,8] => 232332378
[2,3,2,3,3,3,2,8,9] => 232333289
[2,3,2,3,3,3,2,8,8] => 232333288
[2,3,2,3,3,3,2,7,9] => 232333279
[2,3,2,3,3,3,2,7,8] => 232333278
[2,3,2,3,3,3,3,8,9] => 232333389
[2,3,2,3,3,3,3,8,8] => 232333388
[2,3,2,3,3,3,3,7,9] => 232333379
[2,3,2,3,3,3,3,7,8] => 232333378
[2,3,3,2,2,2,2,8,9] => 233222289
[2,3,3,2,2,2,2,8,8] => 233222288
[2,3,3,2,2,2,2,7,9] => 233222279
[2,3,3,2,2,2,2,7,8] => 233222278
[2,3,3,2,2,2,3,8,9] => 233222389
[2,3,3,2,2,2,3,8,8] => 233222388
[2,3,3,2,2,2,3,7,9] => 233222379
[2,3,3,2,2,2,3,7,8] => 233222378
[2,3,3,2,2,3,2,8,9] => 233223289
[2,3,3,2,2,3,2,8,8] => 233223288
[2,3,3,2,2,3,2,7,9] => 233223279
[2,3,3,2,2,3,2,7,8] => 233223278
[2,3,3,2,2,3,3,8,9] => 233223389
[2,3,3,2,2,3,3,8,8] => 233223388
[2,3,3,2,2,3,3,7,9] => 233223379
[2,3,3,2,2,3,3,7,8] => 233223378
[2,3,3,2,3,2,2,8,9] => 233232289
[2,3,3,2,3,2,2,8,8] => 233232288
[2,3,3,2,3,2,2,7,9] => 233232279
[2,3,3,2,3,2,2,7,8] => 233232278
[2,3,3,2,3,2,3,8,9] => 233232389
[2,3,3,2,3,2,3,8,8] => 233232388
[2,3,3,2,3,2,3,7,9] => 233232379
[2,3,3,2,3,2,3,7,8] => 233232378
[2,3,3,2,3,3,2,8,9] => 233233289
[2,3,3,2,3,3,2,8,8] => 233233288
[2,3,3,2,3,3,2,7,9] => 233233279
[2,3,3,2,3,3,2,7,8] => 233233278
[2,3,3,2,3,3,3,8,9] => 233233389
[2,3,3,2,3,3,3,8,8] => 233233388
[2,3,3,2,3,3,3,7,9] => 233233379
[2,3,3,2,3,3,3,7,8] => 233233378
[2,3,3,3,2,2,2,8,9] => 233322289
[2,3,3,3,2,2,2,8,8] => 233322288
[2,3,3,3,2,2,2,7,9] => 233322279
[2,3,3,3,2,2,2,7,8] => 233322278
[2,3,3,3,2,2,3,8,9] => 233322389
[2,3,3,3,2,2,3,8,8] => 233322388
[2,3,3,3,2,2,3,7,9] => 233322379
[2,3,3,3,2,2,3,7,8] => 233322378
[2,3,3,3,2,3,2,8,9] => 233323289
[2,3,3,3,2,3,2,8,8] => 233323288
[2,3,3,3,2,3,2,7,9] => 233323279
[2,3,3,3,2,3,2,7,8] => 233323278
[2,3,3,3,2,3,3,8,9] => 233323389
[2,3,3,3,2,3,3,8,8] => 233323388
[2,3,3,3,2,3,3,7,9] => 233323379
[2,3,3,3,2,3,3,7,8] => 233323378
[2,3,3,3,3,2,2,8,9] => 233332289
[2,3,3,3,3,2,2,8,8] => 233332288
[2,3,3,3,3,2,2,7,9] => 233332279
[2,3,3,3,3,2,2,7,8] => 233332278
[2,3,3,3,3,2,3,8,9] => 233332389
[2,3,3,3,3,2,3,8,8] => 233332388
[2,3,3,3,3,2,3,7,9] => 233332379
[2,3,3,3,3,2,3,7,8] => 233332378
[2,3,3,3,3,3,2,8,9] => 233333289
[2,3,3,3,3,3,2,8,8] => 233333288
[2,3,3,3,3,3,2,7,9] => 233333279
[2,3,3,3,3,3,2,7,8] => 233333278
[2,3,3,3,3,3,3,8,9] => 233333389
[2,3,3,3,3,3,3,8,8] => 233333388
[2,3,3,3,3,3,3,7,9] => 233333379
[2,3,3,3,3,3,3,7,8] => 233333378
